civihr_default_theme/scss/pages/_my-leave-request.scss

```diff
@@ -0,0 +1,19 @@
+.chr_leave-request-form__group {
+  margin-bottom: 15px !important;
```

Comment:
It was mentioned as it was extended in table and table already has `margin-bottom`. Now `!important` moved to `chr_leave-request-form__table` class